README.md

```diff
@@ -25,6 +25,12 @@ or you can directly install it for the current user by running this command:
 python setup.py install
 ```
 
+SASS is required in order to precompile the custom stylesheets:
```

Comment:
It works and has been changed, I wasn't aware of the existence of this pip library.